tests/unit/amo/components/TestAddonMoreInfo.js

```diff
@@ -551,6 +560,87 @@ describe(__filename, () => {
     expect(root.find(AddonAuthorLinks)).toHaveProp('addon', addon);
   });
 
+  it('renders related categories', () => {
```

Comment:
We probably need some test cases for when things go wrong, e.g., there is no category for the add-on, categories for the add-on don't exist, an API error has occured.

Reply:
As suggested, I have added the following tests: 
(1) `does not render related categories when add-on has no category`,
(2) `does not render related categories when categories for add-on do not exist`,
(3) `renders errors when API error occurs`